bait-files/src/main/webapp/review.js

```diff
@@ -4,11 +4,45 @@ function getPanels() {
     fold.addEventListener('click', function() {
       this.classList.toggle('active');
       const panel = this.nextElementSibling;
-      if (panel.style.maxHeight) {          //when accordion button is clicked, if panel is visible closes panel by setting maxHeight to null
+      if (panel.style.maxHeight) {  // when accordion button is clicked, if
+                                    // panel is visible closes panel by setting
+                                    // maxHeight to null
         panel.style.maxHeight = null;
       } else {
-        panel.style.maxHeight = panel.scrollHeight + 'px';  //when accordion button is clicked, if panel is not visible, set maxHeight so panel is displayed
+        panel.style.maxHeight = panel.scrollHeight +
+            'px';  // when accordion button is clicked, if panel is not visible,
+                   // set maxHeight so panel is displayed
       }
     });
   }
 }
+
+function getReview() {
+  const queryString = window.location.search;
+  fetch('/review' + queryString)
+      .then(response => response.json())
+      .then((request) => {
+        document.getElementById('request-id')
+            .appendChild(document.createTextNode(request.requestId));
+        document.getElementById('user-id').appendChild(
+            document.createTextNode(request.userId));
+        document.getElementById('review-status')
+            .appendChild(document.createTextNode(request.status));
+        document.getElementById('submission-date')
+            .appendChild(document.createTextNode(
+                new Date(request.timestamp).toLocaleDateString()));
```

Comment:
I don't see this timestamp field being defined in your Request class.. should you add it there?  Also, there are a few potential meanings for the word "timestamp" (i.e. creation time, update time), so you may want to call the field something like submissionTime or something (judging from your naming of the HTML element's ID).